api/src/helpers/requests.ts

```diff
@@ -1,5 +1,4 @@
 import { StringLiteral } from "../types";
-import { QueryDslQueryContainer } from "@elastic/elasticsearch/lib/api/types";
```

Comment:
do we have no linting in this repo to point unused stuff out 🤔 ? Probs be worth adding ! (not as part of this PR)